cross/arm32_ci_script.sh

```diff
@@ -235,9 +235,10 @@ function cross_build_corefx_with_docker {
     # Cross building corefx with rootfs in Docker
     if [ "$__buildArch" == "armel" ]; then
         __extraCmd="/p:OverridePackageSource=https://tizen.myget.org/F/dotnet-core/api/v3/index.json"
+        __portableLinux="-portable=false"
```

Comment:
Yes build.sh does support this flag and it it is set it essentially stops passing "-portable" to the native builds and set PortableBuilds=false in the manage builds.